onboard/catkin_ws/src/controls/launch/controls.launch

```diff
@@ -7,7 +7,7 @@
         <arg name="sim" value="$(arg sim)" />
     </include>
 
-    <!--node pkg="tf2_ros" type="static_transform_publisher" name="base_link_publisher" args="0 0 0 0 0 -0.7070727 0.7071408 odom base_link" /-->
+    <!--node pkg="tf2_ros" type="static_transform_publisher" name="base_link_publisher" args="0 0 0 0 0 0 1 odom base_link" /-->
```

Comment:
Why this change?

Reply:
reverting a change we'd previously made. that line is commented out just for testing at the moment, and will eventually be parameterized to work with our GUI once we define a better way to test controls.